docs/source/api/standalone.mdx

```diff
@@ -57,7 +57,7 @@ The `startStandaloneServer` function's second optional argument is an object for
 
 An optional asynchronous [`context` initialization function](../data/resolvers#the-context-argument). 
 
-The `context` function should return an object, which gets passed to every resolver that executes for a particular operation. This enables resolvers to share helpful context values, such as a database connection.
+The `context` function should return an object shared across all resolvers that are executing for a particular operation. This enables resolvers to share helpful context values, such as a database connection.
```

Comment:
Same as above

Reply:
Added a suggestion above and will update in both places.